controllers/activities/activities.go

```diff
@@ -51,12 +51,12 @@ func Index(w http.ResponseWriter, r *http.Request, u *mdl.User) error {
 	activities := mdl.FindActivities(c, u, count, page)
 	log.Infof(c, "%s activities = %v", desc, activities)
 
-	json := activitiesToJSON(activities)
+	vm := buildIndexViewModel(activities)
 
-	return templateshlp.RenderJson(w, c, json)
+	return templateshlp.RenderJson(w, c, vm)
 }
 
-func activitiesToJSON(activities []*mdl.Activity) []mdl.ActivityJson {
+func buildIndexViewModel(activities []*mdl.Activity) []mdl.ActivityJson {
```

Comment:
I is because the controller is called `Index` maybe `buildActivitiesIndexViewModel` 

We might add another controller on activities later, so `buildActivitiesViewModel` seems too general